agent-control/src/opamp/remote_config/validators/signature/validator.rs

```diff
@@ -19,7 +19,7 @@ use url::Url;
 
 const DEFAULT_CERTIFICATE_SERVER_URL: &str = "https://newrelic.com/";
 const DEFAULT_PUBLIC_KEY_SERVER_URL: &str =
-    "https://publickeys.newrelic.com/signing/blob-management/GLOBAL/AgentConfiguration";
+    "https://staging-publickeys.newrelic.com/r/blob-management/GLOBAL/AgentConfiguration";
```

Comment:
We are setting the staging one for know to easy testing, right?

Reply:
I'll remove staging, I hope that with fleet today we can get the final URL